lab-1/by.Vladimir_Kondr.quizer/src/core/Pair.java

```diff
@@ -0,0 +1,4 @@
+package core;
+
+public record Pair<K, V>(K key, V value) {
```

Comment:
To be honest, I tried but couldn't find an easy way to import this package, so I decided it would be much easier to just add this record (considering it's the only thing I need from the aforementioned package and it's not that hard to implement :) ).